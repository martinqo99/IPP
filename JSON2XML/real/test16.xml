<?xml version="1.0" encoding="UTF-8"?>

<poslední value="true" />
<další>
  <array>
    <item>10.9</item>
    <item>-0.1</item>
    <item>-1.08</item>
  </array>
</další><xwestxxxxxwestx value="test" />
