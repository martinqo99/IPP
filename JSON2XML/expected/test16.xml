<?xml version="1.0" encoding="UTF-8"?>
<další>
    <array>
        <item>
            10
        </item>
        <item>
            -1
        </item>
        <item>
            -2
        </item>
    </array>
</další>
<xwestxxxxxwestx value="test"/>
<poslední value="true"/>
